schemas/common/any.json

```diff
@@ -0,0 +1,53 @@
+  "$id": "http://jsonlogic.com/schemas/common/any.json",
+
+  "title": "All",
+  "description": "Any valid JSON Logic data source.",
+
+  "anyOf": [
+    { "$ref": "http://jsonlogic.com/schemas/operators/accessor/variable.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/accessor/missing.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/accessor/missingSome.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/operators/arithmetic/add.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/arithmetic/divide.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/arithmetic/modulo.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/arithmetic/multiply.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/arithmetic/substract.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/all.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/filter.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/map.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/merge.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/none.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/reduce.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/array/some.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/and.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/equal.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/if.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/not.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/notEqual.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/notnot.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/or.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/strictEqual.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/logic/strictNotEqual.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/operators/misc/in.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/misc/log.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/misc/method.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/operators/numeric/greater.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/numeric/greaterEqual.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/numeric/less.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/numeric/lessEqual.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/numeric/max.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/numeric/min.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/operators/string/cat.json" },
+    { "$ref": "http://jsonlogic.com/schemas/operators/string/substr.json" },
+
+    { "$ref": "http://jsonlogic.com/schemas/common/all-types.json" }
```

Comment:
This schema is very loose, mainly because the API itself is very forgiving

Reply:
I think in this case a forgiving APi is nice